src/app/language.js

```diff
@@ -23,6 +23,12 @@ var setLanguage = function (preferredLanguage) {
     });
 };
 
+
+App.Localization.nativeName = function (lang) {
+    var codes = App.Localization.langcodes
+    return codes[lang]?codes[lang].nativeName:'No native Name for: '+lang;
+}
```

Comment:
i had that thought, i think instead of null i should return `lang` that would make more sense... (it's used i templates where we really don't want to do err handeling)